core/app/[locale]/(default)/account/(tabs)/wishlists/_components/update-wishlist-form/_actions/update-wishlist.ts

```diff
@@ -0,0 +1,44 @@
+'use server';
+
+import { z } from 'zod';
+
+import { updateWishlist as updateWishlistMutation } from '~/client/mutations/update-wishlist';
+
+const UpdateWishlistSchema = z.object({
+  entityId: z.string(),
+  name: z.string(),
+});
+
+export const updateWishlist = async (formData: FormData) => {
+  const { entityId, name } = UpdateWishlistSchema.parse({
+    entityId: formData.get('entityId'),
+    name: formData.get('name'),
+  });
+
+  const input = {
```

Comment:
just thought to consider, if we don't reuse input variable else where maybe we remove it and define arguments directly on L26?🤔